? mozilla/security/nss/tests/mylog
? mozilla/security/nss/tests/cert/mylog
? mozilla/security/nss/tests/fips/mylog
Index: mozilla/security/nss/tests/common/init.sh
===================================================================
RCS file: /cvsroot/mozilla/security/nss/tests/common/init.sh,v
retrieving revision 1.63
diff -u -u -r1.63 init.sh
--- mozilla/security/nss/tests/common/init.sh	4 Apr 2008 07:39:05 -0000	1.63
+++ mozilla/security/nss/tests/common/init.sh	11 Jun 2008 15:53:21 -0000
@@ -265,6 +265,7 @@
     MAKE=gmake
     $MAKE -v >/dev/null 2>&1 || MAKE=make
     $MAKE -v >/dev/null 2>&1 || { echo "You are missing make."; exit 5; }
+    MAKE="$MAKE --no-print-directory"
 
     DIST=${DIST-${MOZILLA_ROOT}/dist}
     SECURITY_ROOT=${SECURITY_ROOT-${MOZILLA_ROOT}/security/nss}
